src/applications/personalization/profile/components/personal-information/ContactInformationField.jsx

```diff
@@ -417,7 +425,9 @@ export const mapStateToProps = (state, ownProps) => {
   const data = selectVAPContactInfoField(state, fieldName);
   const isEmpty = !data;
   const addressValidationType = selectAddressValidationType(state);
-  const activeEditView = selectCurrentlyOpenEditModal(state);
+  const activeView = selectCurrentlyOpenEditModal(state);
+  const activeEditView = activeView?.includes('remove') ? '' : activeView;
+  const activeRemoveView = activeView?.includes('remove') ? activeView : '';
```

Comment:
I don't _think_ a lot of these changes need to be made. I think that if we left things as
```
activeEditView = selectCurrentlyOpenEditModal(state);
```
then down on line 465 you could just do 
```
showRemoveView: activeEditView === `remove-${fieldName}`,
```